src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__container">
+      <div class="stars stars--0">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--1">
```

Comment:
Please add empty lines between multiline sibling blocks of HTML.

Reply:
Add empty lines between multiline sibling blocks of HTML.